findLongest.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * This method is to find the longest element in a Array
+ *
+ * @since 5.0.0
+ * @category Array
+ * @param {Array} array The array to inspect.
+ * @returns {string} Returns  longest element in a Array
+ * @example
+ *
+ * findLongest(["abc", 'abcd', "abcde"])
+ * // => "abcde"
+ */
+function findLongest(array) {
+  if (!Array.isArray(array)) {
+    throw new TypeError('Expected an Array')
+  }
+  if (!array.length) { return undefined }
+  const len = array.map(item => item.length)
```

Comment:
We might have to omit values that are non-strings.  
Or do you think we should throw an error if the item contains items other than just `strings`?

Reply:
yes,it is better

Reply:
i think throw an error is better..